src/main/java/com/adivinaLaPalabra/squad2/back/AdivinaLaPalabra/controllers/GameController.java

```diff
@@ -1,23 +1,33 @@
 package com.adivinaLaPalabra.squad2.back.AdivinaLaPalabra.controllers;
 
-import com.adivinaLaPalabra.squad2.back.AdivinaLaPalabra.responses.NewGameResponse;
+import com.adivinaLaPalabra.squad2.back.AdivinaLaPalabra.entities.Game;
+import com.adivinaLaPalabra.squad2.back.AdivinaLaPalabra.responses.ErrorResponse;
 import com.adivinaLaPalabra.squad2.back.AdivinaLaPalabra.services.impl.GameServiceImpl;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 @CrossOrigin(origins = "*", maxAge = 3600)
 @RestController
 public class GameController {
-    private static final Logger logger = LogManager.getLogger(DictionaryController.class);
+    
+    private static final Logger logger = LogManager.getLogger(WordController.class);
+
+    @ExceptionHandler({ Exception.class })
+    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+    public ErrorResponse handleDatabaseExceptions(Exception e) {
+        e.printStackTrace();
+        return new ErrorResponse(HttpStatus.INTERNAL_SERVER_ERROR.value(), "Ha habido un fallo al generar la partida, ya se ve lo loser que eres. Recarga anda.");
+    }
 
     @Autowired
     public GameServiceImpl gameService;
 
     @GetMapping("/newGame")
-    public NewGameResponse newGame() {
-        logger.info("Request to newGame");
-        return new NewGameResponse(gameService.newGame());
+    public Game newGame() {
```

Comment:
En esta PR lo podéis dejar así, pero apuntaros por algún sitio que habría que usar DTOs. Aquí estás devolviendo la entidad de base de datos completa, pero realmente habría que devolver una versión limitada con la información relevante para front (o para quien use el API). Por ejemplo: el id de la DB no debería exponerse, especialmente si es secuencial, por temas de seguridad (si puedo adivinar el siguiente número, puedo trastear con otra entidad si no hay control de seguridad).